### PR TITLE
Add some of @jlipps albums

### PR DESCRIPTION
**Pull request by jlipps**
Why not?

### CODE REVIEW

File: app/api/player/$id.mjs [hunk 1]
```diff
@@ -5,9 +5,12 @@ export const get = [async function get (req) {
   const { session: prevSession } = req
   const { id } = req.pathParameters
 
-  // `id` formatted as 'N1-N2' where N1 is the single digit album ID,
```
[the remark this answers is not in the document]
Comment (kristoferjoseph) [reply app/api/player/$id.mjs:8]: Now I am scratching my head looking at the sparse arrray here.

[the remark this answers is not in the document]
Comment (jlipps) [reply app/api/player/$id.mjs:8]: the issue was that only single digit IDs worked, so when i added an album with id `10`, it didn't work anymore.

[the remark this answers is not in the document]
Comment (jlipps) [reply app/api/player/$id.mjs:8]:
```js
const [albumId, , trackId] = id
```

This only works with album/track ids of the form `a-b`, but not `aa-b` or `a-bb`.

File: app/api/player/$id.mjs [hunk 2]
```diff
@@ -5,9 +5,12 @@ export const get = [async function get (req) {
...
-  // and N2 is the single digit track number
-  const [albumId, , trackId] = id
+  // `id` formatted as 'N1-N2' where N1 is the numeric album ID,
+  // and N2 is the numeric track number
+  const [, albumId, trackId] = /([0-9]+)-([0-9]+)/.exec(id)
```
Comment (colepeters) [inline app/api/player/$id.mjs:10]: 🔥 👍🏻

File: app/api/player/$id.mjs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jlipps) [reply app/api/player/$id.mjs:8]: there's no sparse arrays here, just destructuring assignment

[the remark this answers is not in the document]
Comment (colepeters) [reply app/api/player/$id.mjs:8]: Ohh, that clears up my confusion, thanks @jlipps. Yeah, array destructuring can look funky if you're skipping indexes 😄

**Review by colepeters — APPROVED** [review]
Love this, thanks for joining in (and upgrading the path parsing)!

File: app/api/player/$id.mjs (hunk 1 above)
[the remark this answers is not in the document]
Comment (kristoferjoseph) [reply app/api/player/$id.mjs:8]: Sorry that was a bad joke.

A sparse Array is an array missing indexes, that destructuring looks like a sparse Array [ 1, , 3]

File: app/api/player/$id.mjs (hunk 2 above)
Comment (kristoferjoseph) [reply app/api/player/$id.mjs:10]: Apparently we need to make albums with more tracks @colepeters 😂

**Pull request merged**